auto_nag/scripts/phonebook.py

```diff
@@ -35,11 +32,19 @@
 
 
 class PhonebookDirectory():
-
-    def __init__(self, config=CONFIG_JSON):
-        config = json.load(open(config, 'r'))
+    def __init__(self, dryrun=False):
         print "Fetching people from phonebook..."
-        self.people = json.loads(requests.get(PEOPLE_URL, auth=(config['ldap_username'], config['ldap_password'])).content)
+        if dryrun:
+            people_json = (get_project_root_path()
```

Comment:
Could you move the people_json = people_json_file out of the this conditionnal. We would have a single declaration to load the json